contracts/token/ERC721/ERC721Holder.sol

```diff
@@ -4,7 +4,7 @@ import "./ERC721Receiver.sol";
 
 
 contract ERC721Holder is ERC721Receiver {
-  function onERC721Received(address, uint256, bytes) public returns(bytes4) {
+  function onERC721Received(address, address, uint256, bytes) public returns(bytes4) {
```

Comment:
Q: What is the purpose of this file? would ERC721Receiver not be the more expressive implementation?

Reply:
The "Receiver" file is the interface as defined in the standard; the "Holder" contract is the minimal implementation of that interface, so I think we have to leave this as is.

I'm not a huge fan of interfaces not being clearly named as such; I don't particularly like the "Holder" name here, but I don't have a deep enough view of the whole OpenZeppelin stack to guarantee that any change I make would actually be an improvement, so I'll leave this one to others.

Reply:
Ah, I realize that now, yeah. I do think we'd benefit from an `I` prefix being applied to all of the interfaces, but that's a larger change. Cheers